EvoS.Framework/DataAccess/Mongo/TrustWarMongoDao.cs

```diff
@@ -0,0 +1,45 @@
+﻿using EvoS.Framework.DataAccess.Daos;
+using MongoDB.Bson;
+using MongoDB.Driver;
+using static EvoS.Framework.DataAccess.Daos.TrustWarDao;
+
+namespace EvoS.Framework.DataAccess.Mongo
+{
+    public class TrustWarMongoDao : MongoDao<ObjectId, TrustWarDaoEntry>, TrustWarDao
+    {
+        public TrustWarMongoDao() : base(
+            "TrustWar",
+            new CreateIndexModel<TrustWarDaoEntry>(Builders<TrustWarDaoEntry>.IndexKeys
+                .Ascending(msg => msg.Warbotics)
+                .Ascending(msg => msg.Evos)
+                .Ascending(msg => msg.Omni)))
```

Comment:
So you sort entries by faction score but only store one entry as you read the old from the db and modify it to update stats?..

I guess we need a collection for misc stuff (MotD and notifications come to mind)